### ISSUE TITLE
Address Accessibility issues for Screwdriver websites 

### ISSUE THREAD

**Issue by jithine** [screwdriver-cd/screwdriver#2402]
**What happened**:

[Accessibility report](https://github.com/screwdriver-cd/screwdriver/issues/2384#issuecomment-808629700) shows various issues that need to be addressed.

**What you expected to happen**:

Accessibility report should be clean.

**How to reproduce it**:

**Comment by rishu2403** [comment]
Hi Jithin, I was working on fixing some of the issues, I wanted to ask that since primary color "#188fff light blue" is not compatible with minimum contrast ratio requirement set by WCAG, what different color contrast can you suggest? It should meet 4.5:1  foreground to background ratio requirement?
